src/bundle/Resources/public/js/alloyeditor/src/toolbars/config/ez-table-cell.js

```diff
@@ -1,8 +1,25 @@
-import EzConfigTableBase from './base-table';
+export default class EzTableCellConfig {
+    constructor(config) {
+        this.name = 'td';
 
-export default class EzTableCellConfig extends EzConfigTableBase {
```

Comment:
I think it would be good if the `tableCell` and `tableHeader` would still extend the EzConfigTableBase.
You can do something like this to override the buttons array:
```js
constructor(config) {
    super(config);
    
    const editAttributesButton = config.attributes[this.name] || config.classes[this.name] ? `${this.name}edit` : '';

    this.buttons = [
        'ezmoveup',
        'ezmovedown',
        editAttributesButton,
        'tableHeading',
        'ezembedinline',
        'ezanchor',
        'eztablerow',
        'eztablecolumn',
        'eztablecell',
        'eztableremove',
        ...config.extraButtons[this.name],
      ];
}
```

Reply:
@dew326 can you please check https://github.com/ezsystems/ezplatform-admin-ui/pull/1232/commits/3b940b7db53be87c5d2d3bcc8795ae5ee6aaa5a1? Table cells extends basic table config, and table header extends table cell. Please let me know your thoughts on this one.

Reply:
I like this approach.